tests/pos/MergeSort_bag.hs

```diff
@@ -9,6 +9,7 @@ module MergeSort_bag (bag, sort) where
 import qualified Language.Haskell.Liquid.Bag as B
 
 {-@ measure bag @-}
+{-@ bag :: Ord a => [a] -> Bag a @-}
```

Comment:
It is awkward referring to Bag in unqualified form here. It refers to the LH type synonym, not the Haskell type. If we qualify this, LH doesn't seem to pick up the meaning of the LH type synonym.